src/resources/postcss/components/_all.pcss

```diff
@@ -5,3 +5,4 @@
 @import "_tooltip.pcss";
 @import "buttons/_small.pcss";
 @import "buttons/_link.pcss";
+@import "your-tickets/_attendees.pcss";
```

Comment:
I would try to avoid the `your-tickets` term as it won't be generic enough to use it on other places. I'd go with something that refers to an attendees list, and that will make it possible to use it in different locations and limit it to `your-tickets` or anything similar.